app/models/item.rb

```diff
@@ -0,0 +1,6 @@
+class Item < ActiveRecord::Base
+  attr_accessible :name, :price
```

Comment:
Let's move the `attr_accessible` below the relationships and insert a linebreak.